imap_processing/tests/swapi/test_swapi_decom.py

```diff
@@ -2,77 +2,81 @@
 import pytest
 
 from imap_processing import imap_module_directory
-from imap_processing.decom import decom_packets
 from imap_processing.swapi.l1.swapi_l1 import (
     SWAPIAPID,
 )
-from imap_processing.utils import group_by_apid
+from imap_processing.utils import packet_file_to_datasets
+
+
+@pytest.fixture(scope="session")
+def decom_test_science_data(swapi_l0_test_data_path):
```

Comment:
It is interesting that they gave you "derived_values" for the science data, but didn't use any "derived_values" for the housekeeping. That is why we have these two different fixtures here now.

Reply:
yes. That's why I had to add if and else in test but your approach is much better.